modules/tide_landing_page/tide_landing_page.module

```diff
@@ -82,10 +82,8 @@ function tide_landing_page_form_node_form_alter(&$form, FormStateInterface $form
     if (isset($form['title']['widget'][0]['value']['#description'])) {
       $form['title']['widget'][0]['value']['#description'] = t('Include a short unique title for your page and keywords.');
     }
-    if (isset($form["field_landing_page_key_journeys"]["widget"][0]["subform"]["field_paragraph_title"]["widget"][0]["value"]["#default_value"])) {
-      $placeholder_value = $form["field_landing_page_key_journeys"]["widget"][0]["subform"]["field_paragraph_title"]["widget"][0]["value"]["#default_value"];
-      $form["field_landing_page_key_journeys"]["widget"][0]["subform"]["field_paragraph_title"]["widget"][0]["value"]["#placeholder"] = $placeholder_value;
-      unset($form["field_landing_page_key_journeys"]["widget"][0]["subform"]["field_paragraph_title"]["widget"][0]["value"]["#default_value"]);
+    if (isset($form["field_landing_page_key_journeys"]["widget"][0]["subform"]["field_paragraph_title"]["widget"][0])) {
```

Comment:
It's a small thing and I know it was already like this, but we tend to use `'` rather than `"` in this context.

Reply:
hey @anthony-malkoun I have fixed this issue.